project/src/test/java/com/google/sps/data/PlacesScorerImplTest.java

```diff
@@ -37,34 +47,93 @@
         .setPriceLevel(4)
         .setLocation(new LatLng(32.08, 34.78))
         .setGoogleUrl("google.com")
-        .setPlaceId("placeid1")
+        .setPlaceId("placeid")
+        .setName("name")
         .setBusinessStatus(BusinessStatus.OPERATIONAL);
 
     private static final LatLng USER_LOCATION = new LatLng(33.12, 34.56);
-    private static final Place PLACE_1 = PLACE_BUILDER.setName("name1").setRating(3).build();
-    private static final Place PLACE_2 = PLACE_BUILDER.setName("name2").setRating(5).build();
-    private static final ImmutableList<Place> PLACES_TO_SCORE = ImmutableList.of(PLACE_1, PLACE_2);
+    private static final Place PLACE_RATING_3 = PLACE_BUILDER.setRating(3).build();
```

Comment:
I think it would be more readable if you create these places on-demand. 

It's useful to have place_1 and place_2 if these places are arbitrary; But if you have specific logic, sometimes it's better to create the objects in the test so it would be easier for the reader to understand the flow.

Reply:
I agree, it is more readable now